src/Middlewares/CacheResponse.php

```diff
@@ -77,10 +77,22 @@ protected function getReplacers(): Collection
 
     protected function getLifetime(array $args): ?int
     {
-        if (count($args) >= 1 && is_numeric($args[0])) {
+        if (count($args) === 0) {
+            return null;
+        }
+
+        /** @var int|string $lifetime */
+        $lifetime = $args[0];
+
+        if (is_numeric($lifetime)) {
             return (int) $args[0];
         }
 
+        if (is_string($lifetime)) {
+            /* @see http://php.net/manual/en/datetime.formats.relative.php */
+            return now()->modify($lifetime)->diffInSeconds();
```

Comment:
What happens if we use `blablabla` here? Will  Carbon throw a good and readable exception?

Reply:
@freekmurze 
```php
now()->modify('blablabla');
```
throws `\ErrorException` with a message `DateTime::modify(): Failed to parse time string (blablabla) at position 0 (b): The timezone could not be found in the database` (bc \DateTime expects a valid modifier OR timezone).

However I think the main drawback of my proposal is custom delimiter for tags, because we can't use comma with proposed syntax.